src/ai/backend/manager/api/vfolder.py

```diff
@@ -2387,6 +2422,7 @@ def create_app(default_cors_options):
     cors.add(add_route('GET',    r'/{name}/files', list_files))
     cors.add(add_route('POST',   r'/{name}/invite', invite))
     cors.add(add_route('POST',   r'/{name}/leave', leave))
+    cors.add(add_route('POST',   r'/{name}/force-leave', force_leave))
```

Comment:
The `force_leave` handler is almost the same as the existing `leave` handler. How about taking out the common leave login to a separate method?